artemis/utils.py

```diff
@@ -85,6 +86,25 @@ def get_host_from_url(url: str) -> str:
     return host
 
 
+def add_port_to_url(url: str) -> str:
+    url_parsed = urllib.parse.urlparse(url)
+    url_parsed_dict = url_parsed._asdict()
+    if ":" not in url_parsed.netloc:
+        port = getservbyname(url_parsed.scheme)
+        url_parsed_dict["netloc"] = url_parsed_dict["netloc"] + ":" + str(port)
+    return urllib.parse.urlunparse(urllib.parse.ParseResult(**url_parsed_dict))
+
+
+def remove_standard_ports_from_url(url: str) -> str:
+    url_parsed = urllib.parse.urlparse(url)
+    url_parsed_dict = url_parsed._asdict()
+    if url_parsed.scheme == "http" and url_parsed.port == 80:
+        url_parsed_dict["netloc"] = url_parsed_dict["netloc"].split(":")[0]
```

Comment:
just to note: sometimes URL might have credentials (ik, 99.999% won't but), so it might be better to change it to `[-1]`